Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Error.swift

```diff
@@ -231,6 +249,18 @@ extension PathHierarchy.Error {
         }
         return "Replace \(from.singleQuoted) with \(to.singleQuoted)"
     }
+
+    /// Check if exactly one of the given candidate symbols is preferred, because it is not
+    /// disfavored for link resolution and all the other symbols are.
+    /// - Parameters:
+    ///   - candidates: An array of candidate node and disambiguation tuples.
+    /// - Returns: An optional tuple containing the preferred node and its disambiguation string, or nil if there is no preferred node.
+    func preferredCandidate(candidates: [(node: PathHierarchy.Node, disambiguation: String)]) -> (node: PathHierarchy.Node, disambiguation: String)? {
```

Comment:
Since `PathHierarchy.handleCollision(node:remaining:collisions:onlyFindSymbols:rawPathForError:)` does the same thing it might be better to move _this_ to "PathHierarchy+Find" instead of repeating the `collisions.singleMatch` in both places.

Reply:
I did move `preferredCandidate` back to PathHierarchy+Find.swift. However, I didn't take the time to check if other code elsewhere could use it.